docs/setting-up/server/docker.md

```diff
@@ -22,6 +22,55 @@ How to run PMM Server with Docker based on our [Docker image].
 
 ---
 
+You can store data from your PMM in:
+
+1. Volume (Preffered method)
+2. Data container
+3. Host directory
+
+
+### Run Docker with volume
+
+1. Pull the image.
+
+    ```sh
+    docker pull percona/pmm-server:2
+    ```
+
+2. Create a volume:
+
+    ```sh
+    docker volume create pmm-data
+    ```
+
+3. Run the image:
+
+    ```sh
+    docker run --detach --restart always \
+    --publish 443:443 \
+    -v pmm-data:/srv \
+    --name pmm-server \
+    percona/pmm-server:2
+    ```
+4. Change the password for the default `admin` user.
+
+    * For PMM versions 2.27.0 and later:
+
+    ```sh
+    docker exec -t pmm-server change-admin-password <new_password>
+    ```
+
+    * For PMM versions prior to 2.27.0:
+
+        ```sh
+        docker exec -t pmm-server bash -c 'grafana-cli --homepath /usr/share/grafana --configOverrides cfg:default.paths.data=/srv/grafana admin reset-admin-password newpass'
+        ```
```

Comment:
I think we can remove this part because we pull the latest image on previous stage:
```
    docker pull percona/pmm-server:2
```